webapp/packages/plugin-codemirror6/src/ReactCodemirror.tsx

```diff
@@ -168,9 +168,13 @@ export const ReactCodemirror = observer<IReactCodeMirrorProps, IEditorRef>(
 
         let isCursorInDoc = cursor && cursor.anchor > 0 && cursor.anchor < view.state.doc.length;
 
-        if (value !== undefined && value !== view.state.doc.toString()) {
-          transaction.changes = { from: 0, to: view.state.doc.length, insert: value };
-          isCursorInDoc = cursor && cursor.anchor > 0 && cursor.anchor < value.length;
+        if (value !== undefined) {
+          const newText = view.state.toText(value);
+
+          if (!newText.eq(view.state.doc)) {
+            transaction.changes = { from: 0, to: view.state.doc.length, insert: newText };
+            isCursorInDoc = cursor && cursor.anchor > 0 && cursor.anchor < newText.length;
+          }
         }
 
         if (cursor && isCursorInDoc && (view.state.selection.main.anchor !== cursor.anchor || view.state.selection.main.head !== cursor.head)) {
```

Comment:
`if (incomingValue !== undefined && incomingView && incomingValue !== incomingView.state.doc.toString()) {`

I believe this one also might be the case for another similar bug. show we do this consistent to the check you just added?